### ISSUE TITLE
ERD Help!

### ISSUE THREAD

**Issue by ghost** [ga-wdi-exercises/project2#691]
May I have some assistance with this ERD? I feel like I may be missing some thing.

![image](https://cloud.githubusercontent.com/assets/8737117/19397410/c409715e-9215-11e6-99e6-8b927713325f.png)

User Narrative (Draft). A user (either a manager or worker) logs in and sees a list of Projects by name. They can click the project and can log their time worked on the project by using forms to clock in and out. The worker can:
1. **clock in** at the start of the day
2. **clock out** for lunch
3. **clock in** again in the afternoon
4. **clock out** clock out at the end of the day

I will spend the weekend designing the application and have seeds ready by Monday morning.

This function/feature is my MVP

**Comment by nolds9** [comment]
So I guess my confusion lies around the differentiation between a `user` and a `worker`. 
What I might recommend to help getting started, Is two pick one end user - i.e. a manager or an employee, then build an app to meet their needs. 

It's going to be a little more difficult / complex to build an app for different "types" of users. If that's the route you truly want to go down, I'd recommend building out one side first, I.e. only from the perspective of a manager, then adding in user roles later with a gem like CanCanCan

**Comment by ghost** [comment]
Okay! I'll stick with Worker for now. I'm wire-framing and will check in again.

Worker will have less authorizations than Manager.

**Comment by ghost** [comment]
**tl;dr**

My CRUD app is a project management app called Heimdall. A worker logs their hours on a project so a Manager can see total hours worked on a project.
